libs/books/feature/src/lib/total-count/total-count.component.ts

```diff
@@ -1,16 +1,29 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, AfterViewChecked } from '@angular/core';
 import { Store } from '@ngrx/store';
 import { getTotalUnread } from '@tmo/books/data-access';
-
+import { SelectorQuery, SelectorType, setAriaLabel } from '@tmo/shared/utils-accessibility';
 @Component({
   selector: 'tmo-total-count',
   templateUrl: './total-count.component.html',
   styleUrls: ['./total-count.component.scss']
 })
-export class TotalCountComponent implements OnInit {
+export class TotalCountComponent implements OnInit, AfterViewChecked {
   totalUnread$ = this.store.select(getTotalUnread);
 
   constructor(private readonly store: Store) {}
 
   ngOnInit(): void {}
+  
+  ngAfterViewChecked(): void {
+    this.configureAccessibility()
+  }
+
+  configureAccessibility() {
```

Comment:
You don't need to do this kind of DOM manipulation at all.

You can bind to the aria label quite simply as `<my-tag [attr.aria-label]="myvalue">` or simply `<my-tag aria-label="...">` if you're assigning a literal.